Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node
+COPY . /.
+WORKDIR /.
+RUN yarn
+EXPOSE 3000
```

Comment:
Need to look at this, it exposes it on 0.0.0.0 to the world

Reply:
Pulled fixes from Ez Crypto